app/controllers/admin/createEvents.py

```diff
@@ -0,0 +1,37 @@
+from dateutil import parser
+from app.models.event import Event
+from flask import request
+from app.controllers.admin import admin_bp
+from app.logic.adminNewEvent import createNewEvent, setValueForUncheckedBox
+from app.logic.validateNewEvent import validateNewEventData
+from flask import flash, redirect, url_for, g
+
+
+@admin_bp.route('/createEvent', methods=['POST'])
+def createEvent():
+
+    if g.current_user.isCeltsAdmin:
+
+        eventData = request.form.copy() #since request.form returns a immutable dict. we need to copy to change the
+        newEventData= setValueForUncheckedBox(eventData)
+
+        # reforamt date int Y-m-d for the backend
+        newEventData['eventStartDate'] = parser.parse(newEventData['eventStartDate'], dayfirst=True)
+        newEventData['eventEndDate'] = parser.parse(newEventData['eventEndDate'], dayfirst=True)
+
+
+        # function to validate data
+        validNewEventData, eventErrorMessage = validateNewEventData(newEventData)

```

Comment:
variable names could use some work. Booleans should often use `is` in the name. I was confused if validNewEventData was a dictionary of validated data or True or False. Since you're using it in an `if` statement I'd call it `dataIsValid`.

also, `eventErrorMessage` is a bit generic. In this specific case, the error message is about validation, so you could name it `validationErrorMessage` or something like that.